wpilibj/src/test/java/edu/wpi/first/wpilibj/geometry/GeometryTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.wpi.first.wpilibj.geometry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GeometryTest {
+
+  private static double kEpsilon = 1E-9;
+
+  /**
+   * Asserts "val1" is within "eps" of "val2".
+   *
+   * @param val1 First operand in comparison.
+   * @param val2 Second operand in comparison.
+   * @param eps Tolerance for whether values are near to each other.
+   */
+  private static void assertNear(double val1, double val2, double eps) {
+    assertTrue(Math.abs(val1 - val2) <= eps, "Difference between " + val1 + " and " + val2
+        + " is greater than " + eps);
+  }
+
+  @Test
+  void testTransforms() {
+    Pose2d initial = new Pose2d(0.0, 0.0, Rotation2d.fromDegrees(45.0));
+    Pose2d last = new Pose2d(5.0, 5.0, Rotation2d.fromDegrees(45.0));
+
+    Pose2d finalRelativeToInitial = last.relativeTo(initial);
+
+    assertNear(finalRelativeToInitial.getTranslation().getX(), 5.0 * Math.sqrt(2.0), kEpsilon);
+    assertNear(finalRelativeToInitial.getTranslation().getY(), 0.0, kEpsilon);
+    assertNear(finalRelativeToInitial.getRotation().getDegrees(), 0.0, kEpsilon);
```

Comment:
wrap in assert all and replace with assertEquals

Reply:
Wrapped in `assertAll` and replaced with `assertEquals` in 7fc4c4f.